### ISSUE TITLE
LoadInstanceGroupPipeline error

### ISSUE THREAD

**Issue by gbrindisi** [GoogleCloudPlatform/forseti-security#502]
Hey all!
Just got this weird exception when running the LoadInstanceGroupPipeline, can you help out on this?

Version used should be 1.0.2

```
2017-07-21 13:34:58,406 __main__     INFO     _run_pipelines Running pipeline LoadInstanceGroupManagersPipeline
2017-07-21 13:40:31,965 __main__     INFO     _run_pipelines Finished running LoadInstanceGroupManagersPipeline
2017-07-21 13:40:31,966 __main__     INFO     _run_pipelines Running pipeline LoadInstanceGroupsPipeline
Traceback (most recent call last):
  File "/opt/forseti-security/venv/lib/python2.7/site-packages/forseti_security-1.0.2-py2.7.egg/google/cloud/security/inventory/inventory_loader.py", line 31
0, in <module>
    app.run()
  File "/opt/forseti-security/codebase/.eggs/google_apputils-0.4.2-py2.7.egg/google/apputils/app.py", line 238, in run
    return _actual_start()
  File "/opt/forseti-security/codebase/.eggs/google_apputils-0.4.2-py2.7.egg/google/apputils/app.py", line 267, in _actual_start
    really_start()
  File "/opt/forseti-security/codebase/.eggs/google_apputils-0.4.2-py2.7.egg/google/apputils/app.py", line 220, in really_start
    sys.exit(main(argv))
  File "/opt/forseti-security/venv/lib/python2.7/site-packages/forseti_security-1.0.2-py2.7.egg/google/cloud/security/inventory/inventory_loader.py", line 28
0, in main
    run_statuses = _run_pipelines(pipelines)
  File "/opt/forseti-security/venv/lib/python2.7/site-packages/forseti_security-1.0.2-py2.7.egg/google/cloud/security/inventory/inventory_loader.py", line 16
5, in _run_pipelines
    pipeline.run()
  File "/opt/forseti-security/venv/lib/python2.7/site-packages/forseti_security-1.0.2-py2.7.egg/google/cloud/security/inventory/pipelines/load_instance_group
s_pipeline.py", line 92, in run
    igs = self._retrieve()
  File "/opt/forseti-security/venv/lib/python2.7/site-packages/forseti_security-1.0.2-py2.7.egg/google/cloud/security/inventory/pipelines/load_instance_group
s_pipeline.py", line 83, in _retrieve
    project_igs = self.api_client.get_instance_groups(project.id)
  File "/opt/forseti-security/venv/lib/python2.7/site-packages/forseti_security-1.0.2-py2.7.egg/google/cloud/security/common/gcp_api/compute.py", line 218, i
n get_instance_groups
    os.path.basename(instance_group.get('zone')),
  File "/opt/forseti-security/venv/lib/python2.7/posixpath.py", line 114, in basename
    i = p.rfind('/') + 1
AttributeError: 'NoneType' object has no attribute 'rfind'
```

**Comment by blueandgold** [comment]
Looks like the issue is with the data, where ```instance_group.get('zone')``` is ```None```. 

Channeling @matthewg to explain why this is so, and how best to handle.

**Comment by blueandgold** [comment]
@gbrindisi, it might help if you can provide the values for instance_group here, so that it might help matthewg@ to diagnose.  Thanks.

**Comment by blueandgold** [comment]
Fixed by PR #507

**Comment by gbrindisi** [comment]
sorry I'm late!
thanks!
